zmsdldb/src/Zmsdldb/Entity/Authority.php

```
@@ -111,9 +111,9 @@ public function removeLocation($locationId)
      * @param String $serviceCsv
      *            only check for this serviceCsv
      *
-     * @return self
+     * @return bool
      */
-    public function isInServiceList($servicelist = array())
+    public function isInServiceList(mixed $servicelist = array())
```

PHPStan (2.2.8)

[error] PHPDoc tag `@param` references unknown parameter: $serviceCsv


(parameter.notFound)